test/test_helper.rb

```diff
@@ -17,6 +17,7 @@
 require "gds-api-adapters"
 require "govuk_schemas"
 require "climate_control"
+require "byebug"
```

Comment:
👀

Reply:
Is there a better way to do this? I couldn't debug my test without it, thought others would probably like to be able to debug the tests as well

Reply:
Generally we don't commit things like this to main, some people have different local dev setups etc. As long as you can verify everything passes locally, remove anything you have added for local testing and commit. As long as CI passes then all is fine 👍

Reply:
Done

Reply:
Ah yeah, sorry - I assumed it was a mistake rather than a deliberate addition. I wouldn't object to it being added to testhelper deliberately I don't think - just seemed unrelated to the PR